.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
```

Comment:
remove `package-lock.json` from the gitignore

Reply:
thank you, deleted